assets/javascripts/discourse/components/payment-plan.js.es6

```diff
@@ -0,0 +1,21 @@
+import Component from "@ember/component";
+import discourseComputed from "discourse-common/utils/decorators";
+
+export default Component.extend({
+  @discourseComputed("selectedPlan")
+  selected(planId) {
+    return planId === this.plan.id;
+  },
+
+  @discourseComputed("plan.type")
+  recurringPlan(type) {
+    return type === "recurring" ? true : false;
```

Comment:
I would make "recurring" a constant defined above the component definition. Then this line would be
```js
return type === RECURRING // no need for the true : false
```

Reply:
Also a constant for "one_time" to be used below